### DISCUSSION TITLE
please delete

### DISCUSSION THREAD

**Discussion by jonhp** [Bodmer/TFT_eSPI#3487] [Q&A - General]

**Comment by elgerg** [comment]
There are loads.
Ive not come up with any compatibility issues when switching processors so far. My current favourite is this
https://www.aliexpress.us/item/3256806436431838.html?spm=a2g0o.productlist.main.1.3bacWcFCWcFC7P&algo_pvid=5f203e00-f068-4d78-b06a-797156a9cbd5&algo_exp_id=5f203e00-f068-4d78-b06a-797156a9cbd5-0&pdp_npi=4%40dis%21USD%2127.38%2120.38%21%21%21192.26%21143.10%21%40211b6c4f17268569199953015e993e%2112000037852055385%21sea%21US%210%21ABX&curPageLogUid=Znln6uPt8fRL&utparam-url=scene%3Asearch%7Cquery_from%3A

Its a wall "switch" but in essence it's a 480x480 4 inch display with relay(s).

I dont work for ali but I did buy a couple of them.
